src/query/strategies/organization.query.strategy.ts

```diff
@@ -1,26 +1,33 @@
-import { getOctokit } from "@actions/github"
-import { processRestResponse } from "../../process/rest.process"
+import { processResponse } from "../../process/process"
 import { Package, QueryStrategy, RestInput } from "../../types"
+import {GitHub} from "@actions/github/lib/utils";
 
-export default class OrganizationRestQueryStrategy implements QueryStrategy {
+export default class OrganizationQueryStrategy implements QueryStrategy {
+  private octokit: InstanceType<typeof GitHub>;
+  
+  constructor(octokit: InstanceType<typeof GitHub>) {
+    this.octokit = octokit;
+  }
+  
   async queryPackages(input: RestInput): Promise<Package[]> {
     return await Promise.all(
       input.names.map(async (name) => {
         const response = await this.queryPackage(input, name)
 
-        return processRestResponse(name, response)
+        return processResponse(name, response)
       })
     )
   }
 
   private async queryPackage(input: RestInput, name: string) {
     try {
-      return await getOctokit(input.token).rest.packages.getAllPackageVersionsForPackageOwnedByOrg({
+      const params = {
         package_name: name,
         package_type: input.type,
         org: input.organization,
         per_page: 100,
-      })
+      }
+      return await this.octokit.rest.packages.getAllPackageVersionsForPackageOwnedByOrg(params)
```

Comment:
`await` is not needed here (has no effect since we return a `Promise` anyway).

Reply:
Updated